source/module_esolver/esolver_ks_lcao_elec.cpp

```diff
@@ -90,13 +90,17 @@ void ESolver_KS_LCAO<TK, TR>::set_matrix_grid(Record_adj& ra)
                              psi_u,
                              dpsi_u,
                              d2psi_u,
-                             GlobalV::NUM_STREAM);
-    psi_u.clear();
```

Comment:
why did you delete lines realted to psi_u, etc.?

Reply:
`psi_u` is created within this function. Since it is of vector type, it is automatically released when the function ends, so there is no need for manual memory management.